src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const styleLines = sourceString.split(';').filter(line => line.trim());
+
+  styleLines.forEach(styleLine => {
+    const [property, value] = styleLine.split(':').map(part => part.trim());
+
+    if(property, value){
```

Comment:
[CODE KNOWLEDGE] - There seems to be a typo in the condition `if(property, value)`. This is not the correct way to check if both `property` and `value` are present. The comma operator will result in only the last operand (`value`) being evaluated for the truthiness check. You should use a logical AND operator `&&` to check if both `property` and `value` are truthy.